packages/i18n/index.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { t as t_dev_or_prod } from './lib/i18n';
+import { t as t_dev } from './lib/i18n-dev';
+
+export const t = t_dev_or_prod as unknown as typeof t_dev;
```

Comment:
To prevent file changes with the `pnpm build` and `pnpm dev` commands, I have set them to be ignored. Type errors are ignored with comments, while type support remains the same as `i18n-dev.ts`.